src/common/Screen.js

```diff
@@ -5,7 +5,9 @@ import type { Node as React$Node } from 'react';
 import { StyleSheet, View, ScrollView } from 'react-native';
 import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
 
-import type { Context, Dimensions, LocalizableText, Dispatch } from '../types';
+import type { ThemeColors } from '../styles';
```

Comment:
> > Also, I don't think this component is even visible when the user goes into their settings to change the theme.
> ... and in particular, I would really prefer to never have to care whether or not this is true.

This is precisely what I meant by what follows the sentence you've quoted: "Even so, I think this
is the correct way to handle a value that, in theory, could change
at any time." And what prompted me to avoid the latent bug in my own code. (Though, as we've seen, I neglected to fix the pre-existing instances I'd been aware of and forgot about.)

> What you might want to do, for testing purposes, is to set up a temporary component in (e.g.) `AppEventHandlers` which basically just does [...]

Thanks for this idea and example code, I'll use it before I resubmit. 🙂